static/src/stylesheets/_common.scss

```diff
@@ -3,7 +3,8 @@
 @import 'module/_search';
 @import 'module/_search-tool';
 @import 'module/_weather';
-@import 'module/_tabs';
+@import 'module/tabs-default.head';
```

Comment:
Maybe we should move this to `_head.common.scss`?

Reply:
I'm not sure of the file names here, `_tabs-enhanced` and `_tabs-default.head` 

`_tabs-enhanced` implies to me that the styles contained within are for classes added by our enhanced JavaScript. This reflects how `_rich-links-enhanced` is used.

I think it would be clearer to use the name `_tabs.head` and `tabs` to indicate inlined and loaded styles respectively.

Reply:
agreed to all ends, i copied the existing nomenclature blindly!